doc/agent-workflow.md

```diff
@@ -0,0 +1,68 @@
+# stOLAS — Agent Workflow
+
+This file provides agent workflow description section. 
+---
+
+## Workflow Diagram
+
+```mermaid
+graph LR
+  StartRound
+  CheckAnyWorkRound
+  WaitingRound
+
+  StartRound-->|DONE|CheckAnyWorkRound
+  CheckAnyWorkRound-->|CLAIM_BRIDGED_TOKEN|ClaimBridgedTokensRound
+  CheckAnyWorkRound-->|CALL_REDEEM|RedeemRound
+  CheckAnyWorkRound-->|CLAIM_REWARDS|ClaimRewardTokensRound
+  CheckAnyWorkRound-->|CALL_CHECKPOINTS|CheckpointRound 
+  CheckAnyWorkRound-->|TRIGGER_L2_TO_L1|TriggerL2ToL1BridgeRound
+  CheckAnyWorkRound-->|NO_WORK|WaitingRound
+  
+
+  ClaimRewardTokensRound-->|DONE|CheckAnyWorkRound
+  ClaimBridgedTokensRound-->|DONE|CheckAnyWorkRound
+  TriggerL2ToL1BridgeRound-->|DONE|CheckAnyWorkRound
+  CheckpointRound-->|DONE|CheckAnyWorkRound
+  RedeemRound-->|DONE|CheckAnyWorkRound
+  WaitingRound-->|DONE|CheckAnyWorkRound
+```
+
+---
+
+## Agent Workflow — Step by Step
+
+### Claim Bridged Tokens
+
+Agents are going to interact with relevant L1 bridge contracts in order to finalize fund transfers from L2 to L1.
+Each native bridge is specific to its fund transfer finalization times and routines.
```

Comment:
please provide a little more details on the function i need to call on l1 along with the expected call data for example for the gnosis bridge.